pkg/infra/metrics/metrics.go

```diff
@@ -40,6 +40,10 @@ var (
 	M_Aws_CloudWatch_GetMetricData       prometheus.Counter
 	M_DB_DataSource_QueryById            prometheus.Counter
 
+	// LDAPUsersSyncExecutionTime is a metric for
+	// how much time it took to sync the LDAP users
+	LDAPUsersSyncExecutionTime prometheus.Summary
```

Comment:
With latest commits it seems we started to move away from snake case in favor of camel case

Reply:
Yes, looks like.

Reply:
yea, looks like it. Should create a chore/refactor issue to rename the others, looks a bit messy with two naming ways